todo/tests/fixtures/task.py

```
@@ -0,0 +1,37 @@
+from todo.constants.task import TaskPriority
+from todo.models.task import TaskModel
```

Ruff (0.8.2)

`todo.models.task.TaskModel` imported but unused

Remove unused import: `todo.models.task.TaskModel`

(F401)